app/src/main/kotlin/de/devisnik/android/bigmouth/BitesChat.kt

```diff
@@ -190,14 +189,22 @@ class BitesChat : AppCompatActivity(), OnInitListener, Speaker {
         }
     }
 
-    private val audio = getSystemService(Context.AUDIO_SERVICE) as AudioManager
```

Comment:
This crashes since the services are not available before calling `Activity.onCreate`